modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/loading/Load.scala

```diff
@@ -119,14 +120,9 @@ object Load {
       .limitRetries[LoaderAction[F, *]](MaxRetries)
       .join(RetryPolicies.exponentialBackoff(Backoff))
 
-  private def congratulate[F[_]: Monad: Logging: Timer](message: LoaderMessage.ShreddingComplete): LoaderAction[F, Unit] =
-    Timer[F].clock.instantNow.flatMap { now =>
-      val count = message.count.map(c => s"${c.good} events").getOrElse("volume is unknown")
-      val latency = message.timestamps.min match {
-        case Some(earliest) => s"${Duration.between(earliest, now).toSeconds} seconds"
-        case None => "unknown"
-      }
-      Logging[F].info(s"Folder [${message.base}] has been loaded and committed. Success!") *>
-        Logging[F].info(s"Folder [${message.base}] ($count) latency: $latency")
-    }.liftA
+  private def congratulate[F[_]: Logging: Monad](
+    loaded: LoaderMessage.ShreddingComplete
+  ): F[Unit] =
+    Logging[F].info(s"Folder ${loaded.base} loaded successfully") >>
```

Comment:
I thought we agreed to keep logging in place?

Reply:
Yes. I didn't want to compute the metrics twice, so I moved the logging [here](https://github.com/snowplow/snowplow-rdb-loader/pull/391/files#diff-5aefeab53daac289abca1f1ba409202804160cd9fa8996d3016016b86cf302f3R110).